app/models.py

```diff
@@ -113,21 +118,19 @@ def get_lot(self, lot_slug):
         )
 
     def serialize(self):
+        acd = self.application_close_date.strftime(DATE_FORMAT) if self.application_close_date else ''
```

Comment:
This could just be in-lined in the serialisation?  It's only used once, and everything else is in-line, so should be fine?